src/stylesheets/components/_navigation.scss

```diff
@@ -68,33 +44,31 @@
   }
 }
 
-/*--- Navigation Number Controls ---*/
+/*--- Navigation Number ---*/
 
-.navigation-control--left {
-  @include navigation-control(left, - $base-spacing-unit-tiny, $base-spacing-unit-tiny);
-}
-.navigation-control--right {
-  float: none;
-  display: inline-block !important;
-}
+.navigation--number ol {
```

Comment:
In Safari, this is going into a new-line with `display: flex` it's inheriting from the mixin *list--inline*; I found that you can fix this by setting the display property to use *inline-flex* instead, with its parent having `text-align: center;`; this fix is ideal for desktop, for which I think this might be a good solution, which works on all browsers:

```
.navigation--number {
  @include media($desktop) {
    text-align: center;
  }

  ol {
    margin: 0 (-$base-spacing-unit-tiny);

    @include media($desktop) {
      display: inline-flex;
      justify-content: center;
    }
...
```